ls1intum/ArTEMiS
# Feature/integrate new models

### Checklist

### Motivation and Context
We need database changes for new features: notifications, question and answers and lecture support.
In addition, I refactored questions into quiz-questions and fixed some bugs, e.g. that SA questions send the solution as part of the http response to the client before the quiz ends and that the practice mode includes the solution as part of the http response before the quiz ends 

### Description
- [x] Added the entities to artemis.jh for discussion
- [x] Created the Java domain classes (Server)
- [x] Created the liquibase changelog
- [x] Created the TypeScript domain classes (Client)
 
### Steps for Testing
1. Start the server
2. Start the client
3. Check that there are no error messages
